### PR TITLE
Add 126 organizations and 135 sources

### PR DESCRIPTION
**Pull request by filipatfunnel**
October update with new sources and organizations from Funnel.

### CODE REVIEW

File: organizations/2_dehands.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "2dehands",
+  "id": "2DEHANDS",
```
Comment (pfrisella) [inline organizations/2_dehands.json]: ID should match filename. 
Either file should be 2dehands or id renamed to 2_DEHANDS

File: organizations/2_ememain.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "2ememain",
+  "id": "2EMEMAIN",
```
Comment (pfrisella) [inline organizations/2_ememain.json]: ID should match filename.
Either file should be 2ememain or id renamed to 2_EMEMAIN

File: organizations/ad_4_game.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Ad4Game",
+  "id": "AD4GAME",
```
Comment (pfrisella) [inline organizations/ad_4_game.json]: ID should match filename.
Either rename file or ID

File: organizations/adikteev.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Adikteev",
+  "id": "ADIKTEEV",
+  "iconUrl": "https://i1.wp.com/www.adikteev.com/wp-content/uploads/2019/02/cropped-fav.png?fit=32%2C32&ssl=1",
+  "orgUrl": ""
```
Comment (pfrisella) [inline organizations/adikteev.json]: missing org url

File: organizations/go_2_mobi.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "GO2MOBI",
+  "id": "GO2MOBI",
```
Comment (pfrisella) [inline organizations/go_2_mobi.json]: ID should match filename.
Either rename file or ID

File: sources/marketo.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "MARKETO",
+  "name": "Marketo",
+  "categories": ["MARKETING", "TOOLS"],
+  "organization": "MARKETO",
```
Comment (pfrisella) [inline sources/marketo.json]: should this be Adobe?

File: organizations/marketo.json [hunk 7]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/marketo.json]: might not need this org if marketo's org is updated to Adobe

File: organizations/obi_4_wan.json [hunk 8]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "OBI4wan",
+  "id": "OBI4WAN",
```
Comment (pfrisella) [inline organizations/obi_4_wan.json]: ID should match filename.
Either rename file or ID

File: sources/waze.json [hunk 9]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "WAZE",
+  "name": "Waze",
+  "categories": ["TOOLS"],
+  "organization": "WAZE",
```
Comment (pfrisella) [inline sources/waze.json]: Should this be GOOGLE

File: organizations/waze.json [hunk 10]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/waze.json]: you might not need this org if you update waze source to use GOOGLE as org

File: sources/2ememain.json [hunk 11]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "2EMEMAIN",
+  "name": "2ememain",
+  "categories": ["ECOMMERCE"],
+  "organization": "2EMEMAIN",
```
Comment (pfrisella) [inline sources/2ememain.json]: make sure org is consistent with any updates to org entry from my previous comments. Here and below

File: organizations/2_dehands.json (hunk 1 above)
Comment (filipatfunnel) [reply organizations/2_dehands.json]: Thanks for noticing - renamed.

File: organizations/2_ememain.json (hunk 2 above)
Comment (filipatfunnel) [reply organizations/2_ememain.json]: Thanks for noticing - renamed.

File: organizations/ad_4_game.json (hunk 3 above)
Comment (filipatfunnel) [reply organizations/ad_4_game.json]: Thanks for noticing - renamed.

File: organizations/adikteev.json (hunk 4 above)
Comment (filipatfunnel) [reply organizations/adikteev.json]: Thanks for noticing - added organization url

File: organizations/go_2_mobi.json (hunk 5 above)
Comment (filipatfunnel) [reply organizations/go_2_mobi.json]: Thanks for noticing - renamed.

File: sources/marketo.json (hunk 6 above)
Comment (filipatfunnel) [reply sources/marketo.json]: Good catch, changed to Adobe

File: organizations/marketo.json (hunk 7 above)
Comment (filipatfunnel) [reply organizations/marketo.json]: Removed 👍

File: organizations/obi_4_wan.json (hunk 8 above)
Comment (filipatfunnel) [reply organizations/obi_4_wan.json]: Thanks for noticing - renamed.

File: sources/waze.json (hunk 9 above)
Comment (filipatfunnel) [reply sources/waze.json]: Makes sense - updated 👍

File: organizations/waze.json (hunk 10 above)
Comment (filipatfunnel) [reply organizations/waze.json]: Removed

File: sources/2ememain.json (hunk 11 above)
Comment (filipatfunnel) [reply sources/2ememain.json]: Updated filenames, IDs should be consistent 👍

**Comment by filipatfunnel** [comment]
Thank you for the detailed review, much appreciated! Have pushed updates based on your comments. When approved, I'll rebase and rename the commit/PR (as the numbers aren't accurate). Let me know if you have any preference of commit message.

**Comment by filipatfunnel** [comment]
@pfrisella Thanks :) 

I see you merged master into this branch, so I suppose a rebase isn't necessary? Anything else you'd like me to do before this can go out?

**Pull request merged**